python/tvm/driver/tvmc/compiler.py

```diff
@@ -38,7 +38,7 @@
 
 @register_parser
 def add_compile_parser(subparsers):
-    """ Include parser for 'compile' subcommand """
```

Comment:
Seems your format tool has some problem. You shouldn't touch this.

Reply:
I agree. I don't know why this format error is observed on the file untouched by me.
Once reverting the commit, the error bothers CI build again, [https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-8125/4/pipeline](https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-8125/4/pipeline), this is why I touched `python/tvm/driver/tvmc/compiler.py`.

Reply:
Once you rebase this PR with the latest from `main`, you'll get #8194, which fixes all these formatting issues. See that Pr for more context.

Reply:
Thank you @leandron, CI build looks working well.